wagtail/contrib/redirects/management/commands/import_redirects.py

```diff
@@ -0,0 +1,182 @@
+import os
+
+import tablib
+from django.core.management.base import BaseCommand
+from tablib.formats import registry
+from wagtail.contrib.redirects.forms import RedirectForm
+from wagtail.core.models import Site
+
+
+class Command(BaseCommand):
+    help = "Imports redirects from .csv, .xls, .xlsx"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--src", help="Path to file", type=str, required=True,
+        )
+        parser.add_argument(
+            "--site_id", help="The site where redirects will be associated", type=int,
+        )
+        parser.add_argument(
+            "--permanent",
+            help="Save redirects as permanent redirects",
+            type=bool,
+            default=True,
```

Comment:
I recommend that all bool args be converted to optional args.

So instead of having to type
`python manage.py import_redirects --src ./Untitled\ spreadsheet\(1\).xlsx --permanent TRUE --dry_run TRUE`

You can write 
`python manage.py import_redirects --src ./Untitled\ spreadsheet\(1\).xlsx --permanent --dry_run`

Simply excluding the arg will be the false scenario, however as permanent is default true it might make sense to leave this as an arg. Unless the arg becomes `--temporary`, where not including that arg will make the redirect permanent. Your call, as it is probably best for the user to be explicit here. `dry_run` should be an optional arg though.

see - https://docs.djangoproject.com/en/3.0/howto/custom-management-commands/#accepting-optional-arguments